smdebug/core/utils.py

```diff
@@ -22,6 +25,24 @@
 logger = get_logger()
 
 
+def make_numpy_array(x):
+    if isinstance(x, np.ndarray):
+        return x
+    elif np.isscalar(x):
+        return np.array([x])
+    elif isinstance(x, tuple):
+        return np.asarray(x)
+    elif isinstance(x, list):
+        return np.asarray(x)
+    elif isinstance(x, dict):
+        return np.array(x)
+    else:
+        raise TypeError(
+            "_make_numpy_array only accepts input types of numpy.ndarray, scalar,"
```

Comment:
mention that list, dict, mx ndarry, torch tensor is supported? its become quite a big list. see if you can reword so we dont have a list of supported types in the log.